vispy/ipython/widget.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2014, 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+from IPython.html.widgets import DOMWidget
+from IPython.utils.traitlets import Unicode, Int, Bool
+from vispy.app.backends._ipynb_util import create_glir_message
```

Comment:
dont you want
from vispy.ipython import VispyWidget

in your notebook app?

maybe we can create the canvas directly from the widget or even make the
widget derive from the widget ideas?

On Wed, Mar 4, 2015, 15:07 Cyrille Rossant <notifications@github.com> wrote:

> In vispy/ipython/widget.py
> <https://github.com/rossant/vispy/pull/2#discussion_r25776169>:
>
> > @@ -0,0 +1,83 @@
> > +# -*- coding: utf-8 -*-
> > +# Copyright (c) 2014, 2015, Vispy Development Team.
> > +# Distributed under the (new) BSD License. See LICENSE.txt for more info.
> > +
> > +from IPython.html.widgets import DOMWidget
> > +from IPython.utils.traitlets import Unicode, Int, Bool
> > +from vispy.app.backends._ipynb_util import create_glir_message
>
> move _ipynb_util to vispy/app/backends/_ipython/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rossant/vispy/pull/2/files#r25776169>.
>